src/geos_functions.jl

```diff
@@ -1352,7 +1349,7 @@ The precision reduced result. Caller must free with `GEOSGeom_destroy()` NULL on
 function setPrecision(
     geom::GEOSGeom,
     gridSize::Real,
-    flags::GEOSPrecisionRules,
+    flags::Union{GEOSPrecisionRules,Integer},
```

Comment:
Why this change without the check?

Reply:
Because the check disallows bitwise or combined flags, see also the last commit message.

Reply:
Wouldn't those bitflags be an `UInt8` instead of `Integer`?

Reply:
`ccall` will convert any `Integer` to the `Cint` that it ultimately needs. I added tests for both `UInt8` and the enum bitflags, which become `UInt32`. And `Int` was already tested.